test/built-ins/DataView/prototype.js

```diff
@@ -0,0 +1,24 @@
+// Copyright (C) 2016 the V8 project authors. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+esid: sec-dataview.prototype
+es6id: 24.2.3.1
+description: >
+  The initial value of DataView.prototype is the DataView prototype object.
+info: >
+  The initial value of DataView.prototype is the intrinsic object
+  %DataViewPrototype%.
+
+  This property has the attributes { [[Writable]]: false, [[Enumerable]]: false,
+  [[Configurable]]: false }.
+includes: [propertyHelper.js]
+---*/
+
+var sample = new DataView(new ArrayBuffer(1), 0);
+
+assert.sameValue(Object.getPrototypeOf(sample), DataView.prototype);
```

Comment:
This is more a test for [[Construct]] than for DataView. Plus, the same concern is already tested in the file named `return-instance.js` in this patch.